Microsoft.Toolkit.Uwp.SampleApp/SamplePages/RemoteDevicePicker/RemoteDevicePickerControlPage.xaml.cs

```diff
@@ -0,0 +1,35 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Linq;
+using Microsoft.Toolkit.Uwp.UI.Controls;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Input;
+
+namespace Microsoft.Toolkit.Uwp.SampleApp.SamplePages
+{
+    public sealed partial class RemoteDevicePickerControlPage : Page
+    {
+        public RemoteDevicePickerControlPage()
+        {
+            InitializeComponent();
+            var selectionEnum = Enum.GetNames(typeof(ListViewSelectionMode)).Cast<string>().ToList();
```

Comment:
Not sure if None or Extended makes sense for the selection mode. Should we have a new enum that only has Single and Multiple?

Reply:
Makes more sense.

Reply:
Looks like esc to close works when you click in the content area. So maybe set a focus inside the popup?

Reply:
Noticed that. Will commit with focus.